pysrc/bytewax/execution.py

```diff
@@ -3,16 +3,7 @@
 Run an instantiated `bytewax.dataflow.Dataflow` using one of the entry
 point functions in this module.
 """
-
-import datetime
-from typing import Any, Iterable, List, Optional, Tuple
-
-from multiprocess import get_context
-
-from bytewax.dataflow import Dataflow
-from bytewax.recovery import RecoveryConfig
-
-from .bytewax import cluster_main, run_main  # noqa: F401
+from .bytewax import cluster_main, run_main, spawn_cluster  # noqa: F401
```

Comment:
Also I wouldn't mind moving all this directly into `bytewax.run` since it's all used from there. And is shorter to type. I don't think we need a separate mod.

Reply:
`cluster_main` is now in `bytewax.run`, while `run_main` is in `bytewax.testing` to make it clear it should be used in production.